inc/languages/english/admin/config_settings.lang.php

```diff
@@ -90,24 +90,24 @@
 $l['error_ajax_search'] = "There was a problem searching for settings:";
 $l['error_ajax_unknown'] = "An unknown error occurred while searching for settings.";
 $l['error_chmod_settings_file'] = "The settings file \"./inc/settings.php\" isn't writable. Please CHMOD to 666.<br />For more information on CHMODing, see the <a href=\"https://docs.mybb.com/1.8/administration/security/file-permissions\" target=\"_blank\" rel=\"noopener\">MyBB Docs</a>.";
-$l['error_admin_email_settings_empty'] = '<div class="smalltext" style="font-weight: normal;">Please note that the <strong>Admin Email</strong> was not saved as the setting can not be empty, and must be a valid email address.</div>';
+$l['error_admin_email_settings_empty'] = "<div class=\"smalltext\" style=\"font-weight: normal;\">Please note that the <strong>Admin Email</strong> was not saved as the setting can not be empty, and must be a valid email address.</div>";
```

Comment:
I guess, but it is done in this way in many other places in the language files where double quote is used when embedding HTML and thus also escaping a lot of characters.

Reply:
It would certainly be good to have a standard, as the current code has been contributed over many years by many different developers with different approaches.

For an example of this difference in approaches, [I write HTML differently to others](https://github.com/mybb/mybb/pull/4193/files#diff-bad7941be860c04bf31b5406e617741e0d90e0530436b1a1c9917c2be85a83b0R51).

Hopefully others will chime in with their thoughts.

Reply:
I think it is not worth to do the larger part at the cost of comprehension.

Edit: I think, eventually, we should be dropping the style and class elements usage from language strings.